src/executor/execute.rs

```diff
@@ -21,23 +21,36 @@ pub(crate) enum ExecResultHolder {
 // ExecResult //
 
 impl ExecResult {
```

Comment:
Is there a way we can keep `ExecResult` as is? It is just a wrapper over SQLx's QueryResult. We should not be changing ExecResult. We should only change InsertResult.

Reply:
Let me think...